tests/thread_cooperation/tests/01-run.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+# Copyright (C) 2017 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    # 1st 6LoWPAN fragment
+    child.expect(r"\[START\].*\n")
+    child.expect(u"[SUCCESS]")
```

Comment:
As stated [here](https://github.com/RIOT-OS/RIOT/pull/7906#discussion_r148817267), the `u` is useless with python 3.
Another thing: either use ``child.expect_exact` or escace the `[ ]` in the `[SUCCESS]` string. For me, `make test` fails on native because of this.